roles/kubespray-defaults/defaults/main.yaml

```diff
@@ -409,6 +409,7 @@ ingress_alb_enabled: false
 cert_manager_enabled: false
 expand_persistent_volumes: false
 metallb_enabled: false
+metallb_speaker_enabled: true
```

Comment:
In this case(default `true`), metallb_speaker would be downloaded even if metallb is disabled. Right?
I guess it is better to use `metallb_enabled` value for  metallb_speaker_enabled by default.

Reply:
Good catch, you're wright!

I also decided to refactor all `metallb_speaker_enabled` to be set to `metallb_speaker_enabled: "{{ metallb_enabled }}"` since enabling `metallb_speaker_enabled` only make sense when enabling  `metallb_enabled`.

What do you think about it?

Reply:
Thanks for doing that, I think that is a nice idea.

/lgtm